packages/react-router-dom/server.tsx

```diff
@@ -142,6 +147,14 @@ export function StaticRouterProvider({
   );
 }
 
+function DataRoutes({
+  routes,
+}: {
+  routes: DataRouteObject[];
+}): React.ReactElement | null {
+  return useRoutes(routes);
+}
```

Comment:
This allows us to remove the fork from `<Routes>` since we can pass the routes more directly to `useRoutes` this way instead of reaching back into context